src/registrar/assets/js/get-gov.js

```diff
@@ -0,0 +1,169 @@
+
+/**
+ * Helper function. Tracks state of selected radio button.
+ * 
+ * This is required due to JavaScript not having a native
+ * event trigger for "deselect" on radio buttons. Tracking
+ * which button has been deselected (and hiding the associated
+ * toggleable) is a manual task.
+ * */
+function setSelected(radioButton) {
+  selected[radioButton.name] = radioButton;
+}
+
+/** Helper function. Announces changes to assistive technology users. */
+function announce(id, text) {
+  const liveRegion = document.getElementById(id + "-live-region");
+  liveRegion.innerHTML = text;
+}
+
+/** On radio button selection change, handles associated toggleables. */
+function handleToggle(e) {
+  // remove toggleable for deselected radio button
+  const selectionExists = e.target.name in selected;
+
+  if (selectionExists) {
+    const hasToggleable = selected[e.target.name].id in radioToggles;
+    if (hasToggleable) {
+      const prevRadio = selected[e.target.name];
+      const prevToggleable = radioToggles[prevRadio.id];
+  
+      const selectionHasChanged = (e.target != prevRadio);
+      const prevSelectionVisible = (prevToggleable.style.visibility === "visible");
+      if (selectionHasChanged && prevSelectionVisible) {
+        makeHidden(prevToggleable);
+        removeRequired(prevToggleable);
+        announce(prevToggleable.id, ARIA.QUESTION_REMOVED);
+      }
+    }
+  }
+
+  // if the currently selected radio button has a toggle
+  // make it visible
+  if (e.target.id in radioToggles) {
+    const toggleable = radioToggles[e.target.id];
+    setSelected(e.target);
+    markRequired(toggleable);
+    makeVisible(toggleable);
+    announce(toggleable.id, ARIA.QUESTION_ADDED);
+  }
+}
+
+/**
+ * An IIFE that will hide any elements with `hide-on-load` attribute.
+ * 
+ * Why not start with `hidden`? Because this is for use with form questions:
+ * if Javascript fails, users will still need access to those questions.
+ */
+ (function hiddenInit() {
+  "use strict";
+  const hiddens = document.querySelectorAll('[hide-on-load]');
+  for(const hidden of hiddens) {
+    makeHidden(hidden);
+    removeRequired(hidden);
+  }
+})();
+
+/**
+ * An IIFE that adds onChange listeners to radio buttons.
+ * 
+ * An element with `toggle-by="<id>"` will be hidden/shown
+ * by a radio button with `id="<id>"`.
+ * 
+ * It also inserts the ARIA live region to be used when
+ * announcing show/hide to screen reader users.
+ */
+(function toggleInit() {
+  "use strict";
+
+  // get elements to show/hide
+  const toggleables = document.querySelectorAll('[toggle-by]');
+
+  for(const toggleable of toggleables) {
+    // get the (comma-seperated) list of radio button ids
+    // which trigger this toggleable to become visible
+    const attribute = toggleable.getAttribute("toggle-by") || "";
+    if (!attribute.length) continue;
+    const radioIDs = attribute.split(",");
+
+    createLiveRegion(toggleable.id)
+    console.log('toggleable :', toggleable);
```

Comment:
**suggestion**: I think that we shouldn't keep debugging console statements in our Javascript when we get to production.

Reply:
Good catch, thank you!